check for OBJECTID in address objects

jgravois opened the pull request:
closes #90.  now you can supply an objectid two different ways:
```javascript
batch.geocode({
  singleLine: "224 N. College Ave, Bloomington, Indiana"//,
  OBJECTID: 1234
});

// or

batch.geocode("224 N. College Ave, Bloomington, Indiana", 1234);
```
thanks for the catch @wilsaj

wilsaj commented:
Looks good. Minor thing - is an OBJECTID of 0 valid within the geocoder? If it isn't valid then it doesn't matter, but in this case it will be overwritten - because `(!0) === false`

jgravois commented:
good call.  i've fixed and rebased.  can't believe i forgot that (especially after [this](https://github.com/koopjs/koop-ckan/issues/2#issuecomment-67259452)).

@nixta i know you're headed to SOTMUS, but can you give this a quick review and merge whenever you have a minute please?

jgravois commented:
here's my repro case, in case its helpful

```javascript
var Geoservices = require('../geoservices-js');
var client = new Geoservices();

client.authenticate('username', 'pw', { /* optional options */ }, function (err, results) {
  if (err) {
    console.log("ERROR: "+err);
  }
  else {
    var batch = new client.geocode.Batch();
    // confirm that ResultID in response match
      batch.geocode({
        singleLine: "224 N. College Ave, Bloomington, Indiana",
        OBJECTID: i
      });
    }
    // run the batch
    batch.run(function (err, results) {
      console.dir(JSON.stringify(results));
    });
  }
});
```

jgravois commented:
@nixta suggested i might be better off using `hasOwnProperty` (example from esri-leaflet [here](https://github.com/Esri/esri-leaflet/blob/master/src/Request.js#L13), but just for fun i'm going to stick to my guns (since i also found a usage similar to my own [here](https://github.com/Esri/esri-leaflet/blob/master/src/Layers/DynamicMapLayer.js#L18)

@JerrySievert @patrickarlt can one of you chime in and break the tie if you have time?

jgravois commented:
i chatted with @paulcpederson and changed my mind about this check.
